tests/src/test/scala/whisk/core/cli/test/WskBasicUsageTests.scala

```diff
@@ -112,6 +112,41 @@ class WskBasicUsageTests
         }
     }
 
+    // If client certificate verification is off, should ingore run below tests.
+    if (!WhiskProperties.getProperty("whisk.ssl.client.verification").equals("off")){
+        it should "set valid cert key to get expected success result for client certificate verification" in {
+            val tmpwskprops = File.createTempFile("wskprops", ".tmp")
```

Comment:
What is the difference between this test and the ones below which checks property set/unset?